pkg/exprhelpers/exprlib_test.go

```diff
@@ -187,6 +188,48 @@ func TestRegexpInFile(t *testing.T) {
 	}
 }
 
+func TestSpeedMemoized(t *testing.T) {
+	if err := Init(nil); err != nil {
+		log.Fatal(err)
+	}
+	err := FileInit(TestFolder, "bad_user_agents.txt", "regex")
+	if err != nil {
+		log.Fatal(err)
+	}
+	// read lines from tests/all_agents.txt
+	file, err := os.Open("tests/all_agents.txt")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		RegexpInFileMemoized(scanner.Text(), "bad_user_agents.txt")
+	}
+}
+
+func TestSpeed(t *testing.T) {
+	if err := Init(nil); err != nil {
+		log.Fatal(err)
+	}
+	err := FileInit(TestFolder, "bad_user_agents.txt", "regex")
+	if err != nil {
+		log.Fatal(err)
+	}
+	// read lines from tests/all_agents.txt
+	file, err := os.Open("tests/all_agents.txt")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		RegexpInFile(scanner.Text(), "bad_user_agents.txt")
+	}
+}
```

Comment:
Both the tests load the regexes from `bad_user_agents.txt` which is what we use on our scenario. They than call    run their  respective funtions (RegexpInFile and RegexpInFileMemoized) on each line of `all_agents.txt` . 

The `all_agents.txt` is of 5000 lines. Consisting of 1000 unique user agents.